system-parachains/coretime/coretime-polkadot/src/lib.rs

```diff
@@ -118,7 +117,6 @@ pub type UncheckedExtrinsic =
 pub type Migrations = (
 	// permanent
 	pallet_xcm::migration::MigrateToLatestXcmVersion<Runtime>,
-	migrations::FixMigration,
```

Comment:
@seadanda are we ok with removing this or do we still need it?

Reply:
This can be safely removed.